src/System.Diagnostics.Process/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -205,7 +205,14 @@ internal PerformanceMonitor(string machineName)
             private void Init()
             {
 #if FEATURE_REGISTRY
```

Comment:
Incidentally if you want I think you could remove FEATURE_REGISTRY throughout this project and the csproj, since we only compile this code in configurations on Windows. Unix doesn't have Windows perf counters. And on Windows we always have registry. FEATURE_REGISTRY is probably a hangover from Silverlight or suchlike.